db/data_migration/20180906132209_redirect_for_medicines_liaison_group.rb

```diff
@@ -0,0 +1,11 @@
+old_slug = "medicines-advertising-liaison-group"
+new_slug = "medicines-devices-advertising-liaison-group"
+
+group = PolicyGroup.find_by!(slug: old_slug)
+
+Whitehall::SearchIndex.delete(group)
+
+group.update_attributes!(slug: new_slug)
+
+Whitehall::PublishingApi.republish_async(group)
```

Comment:
I'm not sure this is necessary, as Whitehall uses after_commit hooks to trigger Publishing API communication.

Reply:
Ah yes, i forgot about that!

Reply:
Removed 👍